rbc/remotejit.py

```diff
@@ -325,18 +325,19 @@ def describe(self):
         lst = ['']
         fid = 0
         for device, target_info in self.remotejit.targets.items():
-            with target_info:
-                lst.append(f'{device:-^80}')
-                signatures = self.get_signatures()
-                signatures_map = {}
-                for sig in signatures:
-                    fid += 1
-                    signatures_map[fid] = sig
-                llvm_module, succesful_fids = irtools.compile_to_LLVM(
-                    [(self.func, signatures_map)],
-                    target_info,
-                    debug=self.remotejit.debug)
-                lst.append(str(llvm_module))
+            with Type.alias(**self.remotejit.typesystem_aliases):
```

Comment:
This line is required for `print(a_table_function)` to work. Otherwise, the typesystem will raise a [TypeError](https://github.com/xnd-project/rbc/blob/59dcb06ccf8fcfffbd2c0d4e733b67d01062a94c/rbc/typesystem.py#L1536)